### PR TITLE
Moved handling of docker image list to utils

### LINKED ISSUE #360: Synchronize the list of docker images between Submit Notebook and Submit Pipeline
Submit Notebook:
image

Submit pipeline:
image

### PR DESCRIPTION
**Pull request by ajbozarth**
Created a util in application for getting the list of supported docker images
 
Fixes #360 

Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by ajbozarth** [comment]
Opening this as is for feedback and review. I've confirmed this works from the UI side of things but I was having trouble with submissions and don't believe they're caused by my changes. Lots of feedback and review appreciated.

**Comment by ajbozarth** [comment]
For selected I just recreated what was already happening. I figured it would be addressed when we make the notebook submission form persist values (which is an open issue). And as for the first image being the default on first load for both notebooks and pipelines, I also figured that when we switched to a metadata service we would make sure to address that.

File: packages/application/src/services.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  'tensorflow/tensorflow:2.0.0-gpu-py3': 'Tensorflow 2.0 w/ GPU',
+  'pytorch/pytorch:1.2-cuda10.0-cudnn7-runtime': 'Pytorch 1.2 w/ CUDA-runtime',
+  'pytorch/pytorch:1.2-cuda10.0-cudnn7-devel': 'Pytorch 1.2 w/ CUDA-devel',
+  'amancevice/pandas:0.25.0-jupyter': 'Pandas'
+};
+
+/**
+ * A utilities class for various elyra services.
+ */
+export class ElyraServices {
```
Comment (lresende) [inline packages/application/src/services.ts]: I am over the fence if we should call this Elyraxxxx, maybe FrontendServices or ApplicationServices

Comment (ajbozarth) [reply packages/application/src/services.ts]: I have no feeling about the name, I picked it on a whim just to get something down. I think I had though I'd eventually put the service classes here, but later realized they would be better in their own classes/files.

We can pick a name real quick in scrum and I'll update this

**Pull request merged**